Revert https://github.com/emilk/ehttp/commit/884e25a0835a due to

error: Could not find documented features in Cargo.toml
  --> cargo-crates/ehttp-0.3.1/src/lib.rs:21:10
   |
21 | #![doc = document_features::document_features!()]
   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `document_features::document_features` (in Nightly builds, run with -Z macro-backtrace for more info)

--- cargo-crates/ehttp-0.3.1/Cargo.toml.orig	1970-01-01 00:00:01 UTC
+++ cargo-crates/ehttp-0.3.1/Cargo.toml
@@ -43,9 +43,6 @@ all-features = true
 
 [lib]
 
-[dependencies.document-features]
-version = "0.2"
-
 [dependencies.web-sys]
 version = "0.3.52"
 features = [
--- cargo-crates/ehttp-0.3.1/src/lib.rs.orig	1970-01-01 00:00:01 UTC
+++ cargo-crates/ehttp-0.3.1/src/lib.rs
@@ -16,10 +16,6 @@
 //! * [`poll_promise::Promise`](https://docs.rs/poll-promise)
 //! * [`eventuals::Eventual`](https://docs.rs/eventuals/latest/eventuals/struct.Eventual.html)
 //! * [`tokio::sync::watch::channel`](https://docs.rs/tokio/latest/tokio/sync/watch/fn.channel.html)
-//!
-//! ## Feature flags
-#![doc = document_features::document_features!()]
-//!
 
 /// Performs an HTTP request and calls the given callback when done.
 ///
